src/chainparams.cpp

```diff
@@ -55,6 +56,25 @@ static CBlock CreateGenesisBlock(uint32_t nTime, uint32_t nNonce, uint32_t nBits
     return CreateGenesisBlock(pszTimestamp, genesisOutputScript, nTime, nNonce, nBits, nVersion, genesisReward);
 }
 
+/**
+ * Helper for initializing vDeployments via util::ArrayInit
+ */
+static constexpr util::NamedArgument<int, struct bit_arg> _bit;
+static constexpr util::NamedArgument<int64_t, struct start_time_arg> _start_time;
+static constexpr util::NamedArgument<int64_t, struct timeout_arg> _timeout;
+static constexpr util::NamedArgument<int, struct minactheight_arg> _minactheight;
+
+template<size_t O>
```

Comment:
can you use a different name than `O`? I confused it for `0` when first reading

Reply:
Renamed to `Offset`; also, `I` renamed to `Init`